scrape-docs/makeDocs.ts

```diff
@@ -0,0 +1,29 @@
+import scrapeDocs from './src/scrapeDocs';
+import makeDocs from './src/makeRDocs';
+
+const SCRAPED_DOCS_PATH = './fluent-docs.json';
+const DOCS_TEMPLATE_PATH = './documentation_template.R';
+const DOCS_PATH = '../R/documentation.R';
+const REPLACE_MAP = [
+  // Remove trailing whitespaces resulting from scraped docs
+  {
+    regex: /\s+$/gm,
+    replacement: '',
+  },
+  // Add a whitespace between documentation entries, each ends with "NULL"
+  {
+    regex: /NULL/gm,
+    replacement: 'NULL\n',
+  },
+  {
+    regex: /@name pickers/gm,
```

Comment:
The comments for the replacements above are very helpful! Please comment this replacement too :slightly_smiling_face:

Reply:
Done 👌